gpcontrib/arenadata_toolkit/arenadata_toolkit--1.2--1.3.sql

```diff
@@ -0,0 +1,64 @@
+/* gpcontrib/arenadata_toolkit/arenadata_toolkit--1.2--1.3.sql */
+
+/*
+ * Returns columns (table_schema, table_name) ordered by increasing vacuum time.
+ * In this list, if arg is true, then tables that are not yet vacuumed are located first,
+ * and already vacuumed - at the end, else (arg is false) tables that are already
+ * vacuumed are located first, and tables that are not yet vacuumed are located at the end.
+ */
+CREATE FUNCTION arenadata_toolkit.adb_vacuum_strategy_arg(actionname TEXT, arg BOOLEAN)
```

Comment:
what does it mean "arg"? Can we give more informative name for this argument?

Reply:
renamed